src/packagedcode/models.py

```diff
@@ -1530,13 +1530,22 @@ class Package(PackageData):
         label='datasource ids',
         help='List of the datasource ids used to create this package.'
     )
-
+    
+    license_clarity_score = List(
```

Comment:
You have to have one test where the CLI option is enabled and this is present in the package data, otherwise I don't see any proof that this works.

Reply:
Test added please have a look and review